network.go

```diff
@@ -67,6 +67,11 @@ func localIPv4MCastAddrs() ([]string, error) {
 				// Not IPv4.
 				continue
 			}
+			// Fix #33: consider NOT loopback IP addresses
```

Comment:
No need to add comments addressing bugs. That's effectively in the version history and can distract from the code.

Reply:
OK got it. Removed ref to issue